msteams-platform/concepts/device-capabilities/native-device-permissions.md

```diff
@@ -168,14 +168,14 @@ For example:
       /* ... *
     /});
     ```
-# [Desktop](#tab/desktop)
-
-   ![Tabs desktop device permissions prompt](~/assets/images/tabs/device-permissions-prompt.png)
-
 # [Mobile](#tab/mobile)
 
    ![Tabs mobile device permissions prompt](../../assets/images/tabs/MobileLocationPermission.png)
 
+# [Desktop](#tab/desktop)
+
+   ![Tabs desktop device permissions prompt](~/assets/images/tabs/device-permissions-prompt.png)
```

Comment:
Would be great if we can update this image

Reply:
@shuchauh, updated the mobile specific images.